sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 12.8.0-beta.1 (2021-07-26)
+## 12.8.0-beta.1 (2021-07-27)
```

Comment:
Don't need to change date here.

Reply:
Why not? I noticed that we only released on the 28th